ingestion/src/main.rs

```diff
@@ -71,8 +75,28 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     let cancel_token = CancellationToken::new();
     tokio::spawn(util::signal_catcher(cancel_token.clone()));
 
+    // Set up prometheus metrics exporter
+    PrometheusBuilder::new()
+        .set_buckets_for_metric(
+            Matcher::Full("http_requests_duration_seconds".to_string()),
+            &[
+                0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0,
+            ],
+        )
+        .expect("Failed to set metric buckets")
+        .install()
+        .expect("Failed to set up metrics exporter");
+
+    // Register metrics so they're guaranteed to show in exporter output
+    let _ = metrics::histogram!("http_requests_duration_seconds");
+    let _ = metrics::counter!("kldata_messages_received");
+    let _ = metrics::counter!("kldata_failures");
+    let _ = metrics::counter!("kafka_messages_received");
+    let _ = metrics::counter!("kafka_failures");
+    let _ = metrics::counter!("scalar_datapoints");
+    let _ = metrics::counter!("nonscalar_datapoints");
```

Comment:
Should we have a metrics struct that registers these metrics as fields and is passed into the ingestor and kafka tasks (and further down where it's needed)? So we don't have to worry about typos?

Reply:
I prefer @jo-asplin-met-no's idea of `&'static str` constants to avoid typos, since that sticks closer to the usage pattern given by the crate authors, So I'll have a go at implementing that

Reply:
Done

Reply:
Sounds good! It's just a bit ugly having these empty vars in the main file, but eh, we can live with that :)